src/main/java/org/spongepowered/api/item/inventory/Inventory.java

```diff
@@ -472,10 +413,12 @@ default Inventory query(PropertyMatcher<?> propertyMatcher) {
      * in the second one is removed.</p>
      *
      * @param inventory the other inventory
+     *
      * @return an inventory wrapping all slots of both inventories.
      */
     Inventory union(Inventory inventory);
 
```

Comment:
As cyber mentioned, we still need this. containsChild would be a better name.

Reply:
the current impl. (when it works) checks not only for childs but recursivly

Reply:
It scans recursively? Is that how get works? If not, I don't see a problem with containsChild and not analyzing the entire trunk.